app/controllers/PSRController.scala

```diff
@@ -107,8 +108,12 @@ abstract class PSRController extends FrontendBaseController with I18nSupport {
         .map(t => t._2.toString -> t._1)
         .toMap
 
-    def zipWithRefinedIndex[I: Validate[Int, *]]: List[(Refined[Int, I], A)] =
+    // todo: Seems unsafe, indexes that can't be refined get suppressed (they do not get added to the list)
```

Comment:
Is this TODO for later of for this ticket?

Reply:
changed to deprecated

Reply:
changing to deprecated kills the compiler and forces code change, left as todo for now